go.mod

```diff
@@ -24,4 +24,4 @@ require (
 	sigs.k8s.io/testing_frameworks v0.1.1 // indirect
 )
 
-replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190223031456-9a5ae4453bd
+replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190701121157-9a5ae4453bda
```

Comment:
The replace can be removed now, the patches were properly contributed back to goStrongswanVici :)

Reply:
That requires updating the Submariner dependency, is that OK?